src/main/primary_entry/script/page/charts/StatusTotalChart.js

```diff
@@ -60,10 +60,10 @@ export default class StatusTotalChart {
                 .map(a => [Number(a[0]), count += a[1]]);
 
             // Extend chart from 0 to today
-            data.unshift([data[0][0] - 24 * 60 * 60 * 1000, 0]);
             if(data[data.length - 1][0] < today) {
                 data.push([today, data[data.length - 1][1]]);
             }
+            data.unshift([data[0][0] - Math.max((data[1][0] - data[0][0]) / 30, 24 * 60 * 60 * 1000), 0]);
```

Comment:
Moved this line down here in case the line had only one data point. Using the second data point to determine how far back to put the "0" data point. Using `Math.max()` for a minimum of 1 day before the first data point.